### ISSUE TITLE
When install Warning Appears

### ISSUE THREAD

**Issue by jay-shah-rushiinfotech** [yajra/laravel-datatables-buttons#58]
### Code snippet of problem
```
Package phpoffice/phpexcel is abandoned, you should avoid using it. Use phpoffice/phpspreadsheet instead.
```

**It is in package `maatwebsite/excel` ?**

### System details

- Operating System - Ubuntu 16.04 LTS
- PHP Version - 7.2.0
- Laravel Version - 5.5.0
- Laravel-DataTables Version - ^3.1

**Comment by yajra** [comment]
Yes, it is in maatwebsite/excel package.  I think this issue should be raised there but I think are plans to use spreadsheet but haven't check their updates yet.

**Comment by lomotech** [comment]
they are update their version to 3.0 (spreadsheet) but your package still using 2.* (phpexcel)

**Comment by yajra** [comment]
Fixed on v4.0 where excel version is updated to 3.0. Thanks!
